src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs

```diff
@@ -56,5 +56,17 @@ internal class ProjectRestorer : IProjectRestorer
                         result.StartInfo.WorkingDirectory, result.StartInfo.Arguments, result.StdErr, result.StdOut));
             }
         }
+
+        // walk around for https://github.com/dotnet/corefx/issues/26488
+        // fallback osx.10.13 to osx
+        private static string GetRuntimeIdentifierWithMacOsHighSierraFallback()
+        {
+            if (RuntimeEnvironment.GetRuntimeIdentifier() == "osx.10.13-x64")
```

Comment:
is this maybe a little too specific? when 10.14 comes out, we're probably going to have the same problems again..
What about testing for the Mac platform specifically?

Reply:
This is a walkaround. The right fix should be wait until runtime.json have 10.13 in corefx, the same as when 10.14 come out in the future. Also I don’t think restore graph should not be a CLI concern, it should be in corefx. And tests are running in 10.12 machines. I do need to discuss with more people. Abort this walkaround and focus on runtime.json might be the right approach.

Reply:
A problem here would be that the tool would need to be re-published with a new `Microsoft.NETCore.Platforms` package dependency for every new macOS version :(
Or is this dependency in the temp project? That would help.

Reply:
I agree. With OSX in particular, there isn't a good way to downgrade if your backups are gone (that is, after 10.13 was published, all the 10.12 ISOs and install channels were removed).

We need a better story for handling it so that people don't wind up being unable to work after every minor revision change. (CC @KathleenDollard)